cobalt/browser/debug_console.cc

```diff
@@ -205,7 +205,12 @@ bool DebugConsole::FilterOnScreenKeyboardInputEvent(
 
 void DebugConsole::CycleMode() {
   base::AutoLock lock(mode_mutex_);
-  mode_ = (mode_ + 1) % debug::console::kDebugConsoleModeCount;
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kDisableWebDebugger)) {
+    mode_ = (mode_ + 1) % (debug::console::kDebugConsoleModeHud + 1);
```

Comment:
Slight pref for setting an int in the conditional (i.e. num_debuggers) and doing modulo that variable outside of the conditional

Reply:
Agreed. I'll update the CL once the parent merges.

Reply:
Oh, actually it's not possible, the enum is generated from an IDL.

cobalt/debug/console/debug_console_mode.idl

Reply:
Updated as suggested in the parent